lib/matobjplist.gi

```diff
@@ -1173,7 +1251,7 @@ InstallMethod( RankMat, "for a plist matrix",
 
 
 InstallMethodWithRandomSource( Randomize,
-  "for a random source and a mutable plist matrix",
+  "for a random source and a mutable plist vector",
```

Comment:
Why this change? The second argument *is* a plist matrix, not a vector, isn't it?

Reply:
Thanks for spotting this. This change had happened by chance.